### PR TITLE
Migration 007: Set experimental to false

### PR DESCRIPTION
**Pull request by foolip**
This is to bring the data in line with the recently added guideline:
https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#choosing-an-experimental-status

Lots of features which are now widely supported are still marked as
experimental. This migration script conservatively sets experimental to
false for features which are supported in Chrome+Firefox+Safari.

It could later be tweaked to do the same for 2-engine support.

### CODE REVIEW

**Comment by sideshowbarker** [comment]
[X] code reviewed
[X] tested (affects 553 features in 116 files)

So I’d be happy to merge this, but I don’t know who else should also sign off on it — and can’t remember what the next step is after this is actually merged…

**Comment by foolip** [comment]
I also don't know if anyone is responsible for running the migration script after it's merged. I was planning to submit PRs with what it produces, and the edit it to cover 2-engine cases too.

**Comment by foolip** [comment]
I've sent https://github.com/mdn/browser-compat-data/pull/11521 with the changes this script makes.

**Comment by vinyldarkscratch** [comment]
@sideshowbarker The repository owners are the ones who must review and merge PRs involving linters, schema, infrastructure, and scripts like this one.  Our rights as peers doesn't allow us to merge script PRs like this one.

@foolip I don't think we have any particular guideline on who performs the migration PRs, but in the past for the migrations I've written, I've handled the whole process!

**Comment by foolip** [comment]
Ping @ddbeck for review.

**Review by ddbeck — CHANGES_REQUESTED** [review]
This looks good. I have one suggestion.

We do have a doc covering this procedure here: https://github.com/mdn/browser-compat-data/blob/main/docs/migrations.md. The only part we haven't really satisfied yet is a standalone test, however. I think you could extract the for loop to a separate function, write a couple examples (e.g., one showing a feature that needs to be marked false, and another that doesn't), and demonstrate it that way. That would be a nice confidence builder and satisfy our documented expectations.

**Comment by queengooborg** [comment]
@foolip Ping!

**Comment by foolip** [comment]
@queengooborg there we go, test added!

**Comment by queengooborg** [comment]
Quick request: may we actually call this migration 007 instead? We've opened PRs for migrations titled [005](https://github.com/mdn/browser-compat-data/milestone/26) and [006](https://github.com/mdn/browser-compat-data/milestone/27), so this would help us maintain continuity!

(In doing so, you also get to make lots of James Bond jokes.  :P)

**Comment by foolip** [comment]
Sure, let this script be known as James Bond 007: Experimental Migration.

**Comment by foolip** [comment]
@queengooborg is this blocked on anything, or ready to go?

File: scripts/migrations/007-experimental-false.test.js
Comment (queengooborg) [inline scripts/migrations/007-experimental-false.test.js:67]: Shouldn't experimental be set to `false`, since the feature is supported in two different browser engines?

File: scripts/migrations/007-experimental-false.js
Comment (queengooborg) [inline scripts/migrations/007-experimental-false.js:35]: What about mobile browsers?  If a mobile-only feature is supported in both Chrome Android and Safari iOS, shouldn't it also be non-experimental?

**Review by queengooborg — CHANGES_REQUESTED** [review]

File: scripts/migrations/007-experimental-false.test.js
Comment (foolip) [reply scripts/migrations/007-experimental-false.test.js:67]: I have a follow-up to count engines instead, but I'd like to update and use this script in multiple phases, to start with the really simple cases. More scrutiny in review will be needed for the "two different engines" changes I think.

File: scripts/migrations/007-experimental-false.js
Comment (foolip) [reply scripts/migrations/007-experimental-false.js:35]: My follow-up does this.

**Comment by foolip** [comment]
@queengooborg I went to create a follow-up PR, but then thought it's simple enough to just do it right away.

File: scripts/migrations/007-experimental-false.js
Comment (queengooborg) [inline scripts/migrations/007-experimental-false.js:57]: Could we perhaps transform this migration to go both ways, setting `true` and `false`?  

```suggestion
    compat.status.experimental = engineCount < 2;
```

Comment (foolip) [reply scripts/migrations/007-experimental-false.js:57]: That wouldn't match our guidelines:
https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#choosing-an-experimental-status

If we want to infer experimental status entirely by script, we also need to take age into account.

But if we think that is possible/worthwhile, then we should delete the experimental status from source and make it a build step, so not this piece of code.

Comment (queengooborg) [reply scripts/migrations/007-experimental-false.js:57]: That makes sense -- in that case, this is LGTM and we can follow up in the future to add more and potentially automate the experimental status flag!

**Review by queengooborg — APPROVED** [review]

**Pull request merged**
